src/components/Select/index.tsx

```diff
@@ -28,9 +28,11 @@ const StyledGrommetSelect = styled(
 `;
 
 // Set button to occupy parent's width, and a default width.
-const Container = styled.div`
+const Container = styled.div<ContainerProps>`
 	& > button {
 		width: 100%;
+		border-color: ${({ invalid, theme }: ContainerProps) =>
+			invalid ? theme?.colors?.danger.main : 'initial'};
```

Comment:
Ok so `initial` is not the right value here it seems, since it overrides the Select border. Instead if invalid is true, we can return `border-color: theme?.colors?.danger.main`, otherwise we can return `''`.

Reply:
🙏 sorry again @sradevski, I hadn't tested yet. now it should be ok